app/src/main/java/org/mozilla/fenix/home/recenttabs/view/RecentTabs.kt

```diff
@@ -244,25 +267,121 @@ private fun RecentSearchGroupItem(
     }
 }
 
+/**
+ * Displays a recent synced tab.
+ *
+ * @param tab Optional synced tab. If null, displays placeholders.
+ * @param onRecentSyncedTabClick Invoked when item is clicked.
+ * @param onSeeAllClick Invoked when "See all" button is clicked.
+ */
+@Composable
+private fun RecentSyncedTabItem(
+    tab: RecentTab.SyncedTab?,
+    onRecentSyncedTabClick: (RecentTab.SyncedTab) -> Unit,
+    onSeeAllClick: () -> Unit,
+) {
+    val iconPainter = painterResource(id = R.drawable.ic_synced_tabs)
+
+    Card(
+        modifier = Modifier
+            .fillMaxWidth()
+            .requiredHeight(180.dp)
+            .clickable { tab?.let { onRecentSyncedTabClick(tab) } },
+        shape = RoundedCornerShape(8.dp),
+        backgroundColor = FirefoxTheme.colors.layer2,
+        elevation = 6.dp
+    ) {
+        Column(modifier = Modifier.padding(16.dp), verticalArrangement = Arrangement.SpaceBetween) {
+            Row(modifier = Modifier.height(IntrinsicSize.Min)) {
+                if (tab == null) {
+                    RecentTabImagePlaceholder()
+                } else {
+                    RecentTabImage(
+                        tab = tab,
+                        modifier = Modifier
+                            .size(108.dp, 80.dp)
+                            .clip(RoundedCornerShape(8.dp))
+                    )
+                }
+
+                Spacer(modifier = Modifier.width(16.dp))
+
+                Column(
+                    verticalArrangement = Arrangement.SpaceBetween,
+                    modifier = Modifier.fillMaxHeight()
+                ) {
+                    if (tab == null) {
+                        RecentTabTitlePlaceholder()
+                    } else {
+                        RecentTabTitle(title = tab.title)
+                    }
+
+                    Row {
+                        RecentTabIcon(
```

Comment:
The change to `RecentTabIcon` seems to accommodate the drawable icon, but since we aren't needing to load a browser icon and we are just simply displaying a drawable, I think we should just use an `Image()` here to not overload `RecentTabIcon`